src/preferences/dialog/dlgprefkey.cpp

```diff
@@ -265,25 +265,20 @@ void DlgPrefKey::slotUpdate() {
     }
 }
 
-void DlgPrefKey::setNotationCustom(bool active) {
-    if (!active) {
-        return;
-    }
-
+void DlgPrefKey::setNotation(KeyUtils::KeyNotation notation) {
```

Comment:
The whole class and especially these methods are not so great. But at least the handling is consistent now.